geopy/geocoders/osm.py

```diff
@@ -351,9 +356,7 @@ def geocode(
             params['bounded'] = 1
 
         if country_codes is None:
-            country_codes = self.country_bias
-        if not country_codes:
-            country_codes = []
+            country_codes = self.country_bias or []
```

Comment:
Could you please revert this change?

1. It changes behaviour: `country_codes=""` previously was coercing to `[]`, now it would be turning to `[""]`.

2. The previous option was easier to read: at first we fallback to `self.country_bias` if the `country_codes` was not set, then we clean up the value regardless where it came from. These are 2 separate steps.

3. This change is unrelated to the feature you're adding in this PR.

Reply:
Thanks for the feedback and the explanation.
I have reverted the change and just increased the max complexity.